Automattic/jetpack
# Add support for post thumbnail image size on get media endpoint

Fixes [#52010](https://github.com/Automattic/wp-calypso/issues/52010)

#### Changes proposed in this Pull Request:

* This PR adds support for `post-thumbnail` image size on getting media JSON API endpoint. The API endpoint is `/wp/v2/media/{media_id}`. This will force `post-thumbnail` image size on the endpoint which will be picked by

#### Jetpack product discussion


[https://github.com/Automattic/wp-calypso/issues/52010](https://github.com/Automattic/wp-calypso/issues/52010)

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* On a WordPress.com simple/automic site, create a new post
* Open the settings sidebar (by clicking the cog icon in the top right of the editor)
* Select the "Post" tab and scroll to the "Featured Image" section
* Add a Featured Image
* Make sure you see proper sized featured thumbnail image.
